packages/dev-server/package.json

```diff
@@ -38,7 +38,7 @@
     "@expo/bunyan": "4.0.0",
     "@expo/metro-config": "0.3.13",
     "@expo/osascript": "2.0.32",
-    "@react-native-community/cli-server-api": "^5.0.1",
+    "@react-native-community/cli-server-api": "^7.0.3",
```

Comment:
We should drop this dependency and import it from the project if available like we do for metro.